### DISCUSSION TITLE
set default "Expire after" to 30 days when creating link to share

### DISCUSSION THREAD

**Discussion by krzemyczek** [immich-app/immich#11349] [Feature Request]
### The bug

the default one is "never" which in my opinion is a little concerning because: 
1) we can easily forget about the links with photos which we would not want to be public over time
2) after time there could be so many links that it would be impossible to delete all of them
of course you can manually change expiration time but who do that? I almost never because it takes too long when you in hurry. 

### The OS that Immich Server is running on

TrueNAS Scale

### Version of Immich Server

1.109.2/108.0 (?)

### Version of Immich Mobile App

1.109.2

### Platform with the issue

- [ ] Server
- [X] Web
- [X] Mobile

### Your docker-compose.yml content

```YAML
not matter
```

### Your .env content

```Shell
not matter
```

### Reproduction steps

```bash
1. click on photo
2. click "Share"
3. see "Expire after" set to "Never"
```

### Additional information

idk if it's bug or feature request so sorry if wrong place

**Comment by kevincox** [comment]
I would like something similar. I think the default should be configurable per-user. I also try to avoid sharing links that never expire and would like that to require explicit action. (I would probably set my default expiry to 3 months or a year).

**Reply by abamuc** [reply]
IMHO the default should be changeable for the installation, and then again a user could overwrite the default of the installation.

**Comment by eric10k93** [comment]
This is still an issue.  Seems odd that it ever defaulted to "never expire".  The default should definitely have a timeframe.  Ideally this would be configurable, as indicated above.

**Comment by christerdk** [comment]
I think a configurable default timeframe would be very useful and preferred.

**Comment by christerdk** [comment]
Let me just add some perspective to the use case: yes, I am the tech person of our household. I can of course change the expiry dates of link shares. 

But my wife and kids, they wouldn't remember to change the expiry date for each link they create ad-hoc from time to time. 

In other words, it really makes sense to be able to change the default expiry to limited timeframe. In our case "in 1 Year" would probably be the default to minimize friction, but a timeframe is definitely needed to help "clean up" all the links that are getting shared with friends and family.

**Comment by JeromeZephyr** [comment]
+1 on this, it's the decision of the administrator to make sure the users know the default expiration for shared links. 
It could also be a configuration at the user level, and set as default in the admin configuration.

**Comment by N-Soder** [comment]
+1 here, once you have a bunch of users who are not as privacy aware being able to set defaults would be very appreciated - they could obviously change it for what ever reason but as we know most don't so having a logical fall back would be great.
